### PR TITLE
update google docs import

### PR DESCRIPTION
**Pull request by matthewsilver**

### CODE REVIEW

File: serge/configs/caps_ts_config.serge [hunk 1]
```diff
@@ -189,7 +189,7 @@ jobs
             # plugin for more information.
             data
             {
-                path_matches \/(about|title|description)$
+                path_matches \/(content)$
```
Comment (sladymon) [inline serge/configs/caps_ts_config.serge]: Can you change this to:
```path_matches \/(about|title|description|content)$```

**Review by sladymon — COMMENTED** [review]
For the renaming of `/ingestion` to `/import` - do we want that to be just `/import`, or `/import_export`?

Also, I think that some of the Docker stuff references `/ingestion`, so that will likely need to be changed as well for this to still run correctly.

File: serge/configs/caps_ts_config.serge (hunk 1 above)
Comment (matthewsilver) [reply serge/configs/caps_ts_config.serge]: Ah that makes sense, originally I was just doing the google doc words but those other parts would be useful to translate too.

Comment (matthewsilver) [reply serge/configs/caps_ts_config.serge]: Also if that's used for wordpress that's important to keep

**Comment by matthewsilver** [comment]
Good call on renaming other parts Shannon -- I'll fix those and test that all out.

**Review by sladymon — APPROVED** [review]
This runs as expected for the Serge/Translation Service parts. I don't have the credentials to test the Google Doc/WordPress sections, but everything runs fine otherwise.

**Review by sladymon — APPROVED** [review]
These make sense! I think you can go ahead and merge.

File: import_export/import-wordpress.py [hunk 2]
```diff
@@ -11,13 +11,14 @@
...
 
-ROOT_PATH = "/var/tms-data"
+
+ROOT_PATH = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..'))
 
 # TODO: Unit testing, end-to-end testing
 
 def write_post(post):
     title = "wp" + str(post['id'])
-    filename = f"{ROOT_PATH}/source_files/en/{title}.json"
```
Comment (liquidsteves) [inline import_export/import-wordpress.py:20]: You undid some recent fixes to ingest-wordpress when you moved it. Not a big deal, but it's good to look over the final version of a merged diff to make sure there aren't any stray changes.

**Pull request merged**